jaxrs-api/src/main/java/jakarta/ws/rs/ext/EntityPart.java

```diff
@@ -0,0 +1,274 @@
+/*******************************************************************
+* Copyright (c) 2021 Eclipse Foundation
+*
+* This specification document is made available under the terms
+* of the Eclipse Foundation Specification License v1.0, which is
+* available at https://www.eclipse.org/legal/efsl.php.
+*******************************************************************/
+package jakarta.ws.rs.ext;
```

Comment:
I believe this belongs into `.core` package rather than `.ext`, because it models a concept from RFC rather than part of processing pipieline, which is what the rest of the classes in this package represemt.

Reply:
It's a good observation, but likely other things will go into `ext` (extensions) going forward. There's something to be said about keeping the core as small as possible. I believe @andymc12 (correct me if I'm wrong) considered a new package/module, but did not seem worth it.

Reply:
In fact I need to second @pdudits here. This shouldn't be `ext`, it should be `core`.

Reply:
I don't have a strong opinion either way, so I went ahead and moved `EntityPart` to the `core` package.